docs/resources/control-flow/functions/utility-actions.md

```diff
@@ -283,4 +282,378 @@ Follow the steps below to add this action to any widget.
 4. On the right side, search and select the Update App Badge Count (under Alerts/Notifications) action. 
 5. Set the Badget Count Value.
 
-![badge-count.png](img%2Fbadge-count.png)
\ No newline at end of file
+![badge-count.png](img%2Fbadge-count.png)
+
+## Copy to Clipboard
+Using this action, you can allow users to copy a particular text from your app. For example, copying a message or transaction ID and then pasting it into another application.
+
+When this action is triggered, the data is stored temporarily in a special part of the device's memory called the clipboard. The user can then paste the copied text into another application by using the "paste" command.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/R5Qqlm6UXicmBBgwBJH5?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+### Adding Copy to Clipboard [Action]
```

Comment:
For this entire page, I feel it is redundant to add name of action and adding a H3 called "Adding Action[Action]"
Can we just remove all h3 and just keep Action name [Action] and go straight into intro + how to? If you see the summary on the right, it looks very repetitive. If you agree, please change for the entire page.

Reply:
Ya make sense. Updated now!